internal/capability/operand_install.go

```diff
@@ -0,0 +1,68 @@
+package capability
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"opcap/internal/operator"
+	"os"
+	"strings"
+
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func (ca *capAudit) OperandInstall() error {
+	logger.Debugw("installing operand for operator", "package", ca.subscription.Package, "channel", ca.subscription.Channel, "installmode", ca.subscription.InstallModeType)
+
+	var csv *operatorv1alpha1.ClusterServiceVersion
+	ctx := context.Background()
+
+	olmClientset, err := operator.NewOlmClientset()
+	if err != nil {
+		return err
+	}
+
+	opts := v1.ListOptions{}
+	var watch watch.Interface
+	var ok bool
+
+	watch, err = olmClientset.OperatorsV1alpha1().ClusterServiceVersions(ca.namespace).Watch(ctx, opts)
+	if err != nil {
+		return err
+	}
+
+	for event := range watch.ResultChan() {
+		csv, ok = event.Object.(*operatorv1alpha1.ClusterServiceVersion)
+		if !ok {
+			return fmt.Errorf("received unexpected object type from watch: object-type %T", event.Object)
+		}
+		alm := csv.ObjectMeta.Annotations["alm-examples"]
```

Comment:
Another thought that I think we should try to get right from start: integrate alm examples list into the capAudit instance that we're using to run that particular operator. That means operand install doesn't need to find the alm examples. It needs only to install them. That means that we need a separate function that reads alm examples and feeds a new field on the capAudit instance on the newCapAudit method here https://github.com/opdev/opcap/blob/cb1db64e3eff18e83f46204d836eb78c51c2430c/internal/capability/audit.go#L39-L51

After that this function that receives capAudit can access it for each operator by reading the new field on the ca variable.